ci/tasks/build-vps-candidate.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+
+semver=`cat version-semver/number`
+
+pushd vps-release
+
+  mkdir -p blobs/golang_1.7/
```

Comment:
We should always create params for dependencies since they are bound to change. For instance from Golang 1.7.x to 1.8 etc.